yml/hsc.yaml

```diff
@@ -35,28 +35,31 @@ tables:
     mysql:datatype: DOUBLE
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec
+  - name: skymap
+    "@id": "#Object.skymap"
+    datatype: char
+    length: 6
+    mysql:datatype: CHAR(6)
   - name: tractId
     "@id": "#Object.tractId"
     datatype: long
     description: tract ID
     mysql:datatype: BIGINT
   - name: patchId
     "@id": "#Object.patchId"
-    datatype: text
-    length: 3
+    datatype: long
     description: patch ID
-    mysql:datatype: TEXT
+    mysql:datatype: BIGINT
```

Comment:
PatchIds, tractIds, and detectorIds are small human manageable integers (unlike sourceIds)  
4 byte int, INTEGER is prob sufficient for all 3 unless you know something I don't

Reply:
I also think `INTEGER` should be sufficient for all 3, but I see these columns have the type of `BIGINT` in the output parquet files. 

Since we aren't checking the data types in CI and for now we'd just force the types before the database ingest, we may also set this to be what we want (`INTEGER`) rather than what the parquet files have (`BIGINT`).  Do you have a preference?

Reply:
I'm changing those to `INTEGER` despite of their types in the output parquet files